pkg/i18n/dutch.go

```diff
@@ -55,6 +55,9 @@ func addDutch(i18nObject *i18n.Bundle) error {
 		}, &i18n.Message{
 			ID:    "CredentialsPassword",
 			Other: "Wachtwoord",
+		}, &i18n.Message{
+			ID:    "CredentialsPassphrase",
+			Other: "Voer wachtwoordzin in voor SSH-sleutel",
```

Comment:
```suggestion
			Other: "Voer wachtwoord in voor SSH-sleutel",
```
I think this is a better sentence